engine/src/evm/retry_rpc.rs

```diff
@@ -380,7 +380,9 @@ impl<Rpc: EvmSigningRpcApi> EvmRetrySigningRpcApi for EvmRetryRpcClient<Rpc> {
 							Some(gas_limit) =>
 								if estimated_gas > gas_limit {
 									return Err(anyhow::anyhow!(
-										"Estimated gas is greater than the gas limit"
+										"Estimated gas ({}) is greater than the gas limit ({})",
+										estimated_gas,
+										gas_limit
```

Comment:
I added this for debugging and I think it's useful to have when CCM starts being used. I assume even if we release this change by itself we need some engine update so making this change in the engine doesn't introduce any new overhead.